src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -19,6 +19,7 @@ internal sealed class DynamicAnalysisInjector : CompoundInstrumenter
         private readonly MethodSymbol _method;
         private readonly BoundStatement _methodBody;
         private readonly MethodSymbol _createPayload;
+        private readonly PooledHashSet<DebugSourceDocument> _documentsBuilder;
```

Comment:
Is the `_documentsBuilder` field needed, or could it be calculated from the documents referenced in `_spansBuilder` when it's needed, in `CreateBlockPrologue`?